Multiple systems as subfolders (#105)
icpc/ccs-specs: Contest_Archive_Format.md

which case the URLs must be valid. This could be useful in some cases where
the size of the archive matters.

### Multiple systems

Some contests are run with multiple systems that provide a Contest API.
One example of this is a contest run with a primary and
[shadow](ccs_system_requirements#shadow-mode) CCS. Typically in these cases
there is relationship between the systems or a lot of common data, so it
makes sense to merge the data into a single archive.

Data from the primary system (usually the primary CCS) should continue to
be stored as described in this document. Each additional system should
store its data using the same format but in a unique sub-folder

> `systems/<system>`, named after the system that produced it.

Reviewer: Maybe `other-systems`? I'm not sure it's better, but it does make it more explicit that the root directory already contains a full archive and that these are additional ones.

Author: I don't love this, but the only downside I could come up with 'it's longer' so I made the change. :)

Using this pattern means that tools that work against a contest archive
can be run without change on either the primary contest archive or the
archives of any of the other systems.

When there is duplicate data (or the differences between systems are
irrelevant), a system's data can be deleted and replaced with symlinks to

> the data in the parent folder. Using this method reduces the overall size

Reviewer: Nitpick: `parent` is not exactly correct, so I'd suggest:
```suggestion
the data from the primary system. Using this method reduces the overall size
```
